src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1091,7 +1091,11 @@ void RaftPart::processAppendLogResponses(const AppendLogResponses& resps,
               [self = shared_from_this(), term = term_] { self->onLeaderReady(term); });
         }
       } else {
-        LOG(FATAL) << idStr_ << "Failed to commit logs";
+        LOG(DFATAL) << idStr_ << "Failed to commit logs";
+        checkAppendLogResult(code);
+        VLOG(3) << idStr_ << "commitLogs failed";
+        iter.commit(code);
```

Comment:
Why add these three lines here? Why not simply return?

Reply:
Like here https://github.com/vesoft-inc/nebula/pull/5078/files/ddbb5a96236c6753494f533c74044b1dc8d33960#diff-f56618b12fe75ed2289df1551818fdda84ea75d2f5c8b4c406fc196e06e7768bL1062-L1067 

I think it's necessary to commit the error code too.

Reply:
@xtcyclist Of course, you are more familiar with it. If you think it is unnecessary, I can change it to return directly.

Reply:
Better remove these three lines. I think we'd better not adding new codes here.